tikzplotlib/_line2d.py

```diff
@@ -140,12 +140,10 @@ def _marker(
 
     mark_size = obj.get_markersize()
     if mark_size:
+        ff = data["float format"]
```

Comment:
Future work - `float format` should not include the `{:` or `}` IMO - as right now, it cannot be passed to `format`

Reply:
D'accord. I'll take care of that.

Reply:
Another approach would be to make `float format` be a function rather than a format strnig, which could default to `'{:.f}'.format` or similar. What would be nice to eliminate is the two-phase formatting dance.

Reply:
I think something like
```python
ff = data["float format"]
f"point meta min={limits[0]:{ff}}"
```
works for us.

Reply:
If you had a function, that would be `f"point meta min={ff(limits[0])}", which is one character shorter and gives a lot more freedom in the formatting.

Reply:
I have the feeling that admitting a function would be a little over-engineered here. So far, the need for this hasn't crept up. (I'm willing to change my mind of course.) I've now implemented the above idea, let's see how it performs irl.